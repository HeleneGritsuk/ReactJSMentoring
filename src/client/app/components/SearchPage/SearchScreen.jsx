import React from 'react';
import { render } from 'react-dom';
import Footer  from '../Footer/Footer.jsx';
import SearchContainer  from './SearchContainer.jsx';
import SearchResultsContainer  from './SearchResultsContainer.jsx';

class SearchScreen extends React.Component {
  constructor(props) {
    super(props);

  }
  render () {
    return(
      <div className='main-wrapper'>
        <SearchContainer/>
        <SearchResultsContainer/>
        <Footer/>
      </div>
    );
  }
}

export default SearchScreen;
