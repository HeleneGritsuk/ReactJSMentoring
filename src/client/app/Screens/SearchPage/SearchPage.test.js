import React from 'react';
import SearchScreen from './index';
import renderer from 'react-test-renderer';


test('SearchScreen should render index page correctly', () => {
  const component = renderer.create(
    <SearchScreen/>,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
