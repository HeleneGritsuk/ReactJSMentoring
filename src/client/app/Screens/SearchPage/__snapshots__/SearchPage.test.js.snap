// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchScreen should render index page correctly 1`] = `
<div
  className="main-wrapper searchPage"
>
  <div
    className="search"
  >
    <div
      className="container"
    >
      <header>
        <h1>
          netflixroulette
        </h1>
      </header>
      <div
        className="searchForm"
      >
        <h3>
          Find your movie
        </h3>
        <input
          className="searchForm__input"
          type="text"
        />
        <div
          className="searchForm__footer"
        >
          <div
            className="searchForm__searchCriteria"
          >
            <span>
              Search by
            </span>
            <button
              className="active searchForm__searchCriteriaBtns"
            >
              Title
            </button>
            <button
              className="searchForm__searchCriteriaBtns"
            >
              Genre
            </button>
          </div>
          <button
            className="searchForm__searchBtn"
          >
            Search
          </button>
        </div>
      </div>
    </div>
  </div>
  <section
    className="searchResults"
  >
    <div
      className="container"
    >
      <div
        className="searchResults__header"
      >
        <div
          className="searchResults__foundNumber"
        >
          7
           movies found
        </div>
        <div
          className="searchResults__sorting"
        >
          Sort by
          <button
            onClick={[Function]}
          >
             release date
          </button>
          <button
            onClick={[Function]}
          >
             rating
          </button>
        </div>
      </div>
      <div
        className="searchResults__content"
      >
        <div
          className="filmItem"
        >
          <img
            src="http://www.imfdb.org/images/thumb/6/65/Killbillposter.jpg/300px-Killbillposter.jpg"
          />
          <div
            className="filmItem__info"
          >
            <div
              className="filmItem__title"
            >
              Kill Bill
            </div>
            <div
              className="filmItem__year"
            >
              2014
            </div>
          </div>
          <div
            className="filmItem__genre"
          >
            drama
          </div>
        </div>
        <div
          className="filmItem"
        >
          <img
            src="http://www.imfdb.org/images/thumb/6/65/Killbillposter.jpg/300px-Killbillposter.jpg"
          />
          <div
            className="filmItem__info"
          >
            <div
              className="filmItem__title"
            >
              Kill Bill2
            </div>
            <div
              className="filmItem__year"
            >
              2017
            </div>
          </div>
          <div
            className="filmItem__genre"
          >
            adventure
          </div>
        </div>
      </div>
    </div>
  </section>
  <footer>
    <div
      className="container"
    >
      <p>
        netflixroulette
      </p>
    </div>
  </footer>
</div>
`;
